tutorials/W1D1_Instructions/W1D1_Tutorial3.ipynb

```diff
@@ -0,0 +1,515 @@
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e79e6195",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "# Tutorial 3: Content Enhancements\n",
+    "\n",
+    "**Week 1, Day 1: Instructions**\n",
+    "\n",
+    "**By Neuromatch**\n",
+    "\n",
+    "__Content creators:__ Konstantine Tsafatinos\n",
+    "\n",
+    "__Content reviewers:__ Konstantine Tsafatinos\n",
+    "\n",
+    "__Production editors:__ Konstantine Tsafatinos\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "Acknowledgments: [Ella Batty, Spiros Chavlis and neuromatch]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e1685a5",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "___\n",
+    "\n",
+    "# Tutorial Objectives\n",
+    "\n",
+    "*Estimated timing of tutorial: [15 mins]*\n",
+    "\n",
+    "In this tutorial, we'll dive into enhancing your content by integrating various media and interactive elements.\n",
+    "\n",
+    "By the end of this tutorial, you will:\n",
+    "\n",
+    "- learn how to add Airtable links\n",
+    "- learn how to embed videos \n",
+    "- learn how to integrate Kaggle/Collab Buttons\n",
+    "- learn how to upload data to OSF and access it from Jupyter Books"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e6328a8",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "---\n",
+    "\n",
+    "# Section 1: How to add Airtable links\n",
+    " "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "29492952",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "Airtable links should be added at the end of the OUTRO page of each tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "599e8aa1",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "Steps:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d99d752c",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "1. Request the Airtable Link\n",
+    "2. Add Image button\n",
+    "3. Embed and test the button"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60cc05b6",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "**Request the Airtable Link**\n",
+    "\n",
+    "\n",
+    "Reach Out to Neuromatch Staff: Request the specific Airtable link for the day. You will receive a link that resembles:\n",
+    "https://portal.neuromatchacademy.org/api/redirect/to/48f385bc-3cd4-4516-96bb-46bdbdccbe9a"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1e320ad",
+   "metadata": {
+    "execution": {}
+   },
+   "source": [
+    "**Add the Image Button**\n",
+    "\n",
+    "The button image is stored in the \"static\" folder within \"tutorials\" and is named \"AirtableSubmissionButton.png\". You can access it on GitHub using this URL:\n",
+    "\n",
+    "https://github.com/NeuromatchAcademy/course-content/blob/main/tutorials/static/button.png?raw=1\n",
+    "\n",
+    "**Note:** Please refrain from altering this image."
```

Comment:
this link should point to the directory in the repo of the template, not in for the comp neuro course. It may be worth adding an extra explanation about the link. ie. you should be pointing to 
```
https://github.com/neuromatch/course-content-template/blob/main/tutorials/static/button.png?raw=1
```
make sure our template has the image in the static folder and the name is correct. so not button.png, unless thats the actual name of the image